File: OOAD_HW1_Adam_Austin/src/com/company/Shape.java
package com.company;

public class Shape {
    protected String name;
    protected int area;

    Shape() {
        this.name = "Unknown";
    }

    public void printShape() {

        System.out.println(this.name + " of area " + this.area);
    }

    public String getName() {
        return this.name;
    }

    public void setName(String _name) {
        this.name = _name;
    }

    public int getArea() {
        return this.area;
    }
}

File: OOAD_HW2_AdamAustin/Shape.rb
class Shape

    def initialize
        @name = "Unkown"
        @area = -1
    end
    
    def setName(_name)
        @name = _name
    end

    def getName
        return @name
    end

    def getArea
        return @area
    end

    def printShape
        puts "Name: #{@name}, Area: #{@area}"
    end

end
